index.html

```diff
@@ -677,10 +677,8 @@ <h2><code>application/vc+jwt</code></h2>
       <h2><code>application/vc+ld+jwt</code></h2>
       <p>
         This specification registers the
-        <code>application/vc+ld+jwt</code> Media Type specifically for
-        identifying a <a data-cite="rfc7519#section-3">JWT</a>
-        conforming to the Verifiable Credentials JWT format in the `typ`
-        header.
+        <code>application/vc+ld+jwt</code> Media Type for
```

Comment:
```suggestion
        <code>application/vc+ld+json+jwt</code> Media Type for
```

Reply:
Assuming we hear confirmation that this is required from IETF, 

- https://mailarchive.ietf.org/arch/msg/media-types/JxzT03Dhe7Nt8cPAfjbDx3WVQRM/

I will merge this change.